stl/inc/xmemory

```diff
@@ -2155,6 +2184,43 @@ _NODISCARD constexpr auto uses_allocator_construction_args(const _Alloc& _Al, pa
         _STD uses_allocator_construction_args<typename _Ty::second_type>(_Al, _STD get<1>(_STD move(_Pair))));
 }
 
+template <class _Ty, class _Alloc, class _Uty,
+    enable_if_t<_Is_specialization_v<_Ty, pair> && _Is_not_deducible_as_pair<_Uty&>, int>>
+_NODISCARD constexpr auto uses_allocator_construction_args(const _Alloc& _Al, _Uty&& _Ux) noexcept {
+    struct _Pair_remaker {
+        const _Alloc& _Al;
+        _Uty& _Ux;
+
+        constexpr operator remove_cv_t<_Ty>() const {
+            using _Pair_t = remove_cv_t<_Ty>;

```

Comment:
Never mind - this warns `error: unused type alias '_Pair_t' [-Werror,-Wunused-local-typedef]`. (I initially missed this because I usually run `set_environment` before testing, which has the side effect of suppressing Clang warnings in "system headers", oops.)